cclib/parser/turbomoleparser.py

```diff
@@ -819,12 +823,160 @@ def extract(self, inputfile, line):
                         mp2energy = [utils.convertor(utils.float(line.split()[3]), 'hartree', 'eV')]
                         self.append_attribute('mpenergies', mp2energy)
                 line = next(inputfile)
+        
+        
+        # Excited state info from escf.
+        #                          1 singlet a excitation
+        # 
+        # 
+        #  Total energy:                           -112.9060086800507    
+        # 
+        #  Excitation energy:                      0.3111453714493050    
+        # 
+        #  Excitation energy / eV:                  8.466699968968847    
+        # 
+        #  Excitation energy / nm:                  146.4375074832943    
+        # 
+        #  Excitation energy / cm^(-1):             68288.51549285055    
+        # 
+        # 
+        #  Oscillator strength:
+        # 
+        #     velocity representation:             0.1011193926229111    
+        # 
+        #     length representation:               0.9461905579062439E-01
+        # 
+        #     mixed representation:                0.9781524141002400E-01
+        # 
+        # 
+        #  Rotatory strength:
+        # 
+        #     velocity representation:             0.1282566570726007E-10
+        # 
+        #     velocity rep. / 10^(-40)erg*cm^3:    0.8285997783054736E-06
+        # 
+        #     length representation:               0.1242930160103103E-10
+        # 
+        #     length rep. / 10^(-40)erg*cm^3:      0.8029927479925187E-06
+        # 
+        # 
+        #  Dominant contributions:
+        # 
+        #       occ. orbital   energy / eV   virt. orbital     energy / eV   |coeff.|^2*100
+        #         7 a             -10.76           9 a              -0.78       96.7
+        #
+        ## For UHF:
+        #       occ. orbital   energy / eV   virt. orbital     energy / eV   |coeff.|^2*100
+        #         7 a   alpha          -15.12     12 a   alpha            4.74       24.5
+        #         7 a   beta           -15.12     12 a   beta             4.74       24.5        
+        if "Excitation energy:" in line and "excitation" in self.last_lines[-5].split():
+            # The irrep of the state is a few lines back.
+            symm_parts = self.last_lines[-5].split()
+            # We don't always have the multiplicity available.
+            if len(symm_parts) < 4:
+                # No mult.
+                mult = "???"
+            else:
+                mult = symm_parts[1].capitalize()
+            
+            symmetry = "{}-{}".format(mult, symm_parts[-2].capitalize())

```

Comment:
Is http://cclib.github.io/data_notes.html#etsyms a free-for-all? I'm assuming yes and that this is within requirements.

Reply:
I believe so yes, but I'm no expert. The format used here matches that of the Gaussian parser...